src/styles/blocks/nav.scss

```
@@ -0,0 +1,42 @@
+.nav {
+  
+  &__list {
+    display: flex;
+    list-style: none;
+    margin: 0;
+    padding: 0;
```

Redudant, already reseted
```suggestion
    list-style: none;
```

---

Thanks for checking, I'll change it